docs/kb/semgrep-supply-chain/why-no-findings.md

```diff
@@ -0,0 +1,89 @@
+---
+description: Troubleshoot why findings for Semgrep Supply Chain are not showing.
+tags:
+  - Semgrep Supply Chain
+---
+# Why aren't Supply Chain findings showing?
+
+## Ensure compatibility
+
+First, verify that your repository meets the basic requirements for Semgrep Supply Chain:
+
+### Are the language and ecosystem supported?
+
+Check the [Supported Languages table](/docs/supported-languages/#general-availability) for support for the project's language and ecosystem, as well as any ecosystem-specific requirements.
+
+###  Is the supported lockfile present in an appropriate location with the expected name?
+
+Semgrep Supply Chain searches the parent directories of any files for a language for the nearest relevant lockfile. This typically means that monolithic repositories (monorepos) are recognized by the presence of the lockfiles in subdirectories.
```

Comment:
I'm a little confused at the overall message here. Should we be mentioning the appropriate place for lockfiles to be placed?